draft-ietf-lake-reqs.md

```diff
@@ -186,24 +185,26 @@ Moreover, it shall be possible for the receiving endpoint to detect a replayed A
 
 Furthermore, the endpoints shall be able to verify that the identity of the other endpoint is an acceptable identity that it is intended to authenticate to. (This requirement extends beyond the AKE in that the application must enable access to information about acceptable identities without compromising the overall lightweightness of the AKE.)
 
+## Confidentiality {#confidentiality}
 
-## Crypto Agility and Security Properties {#crypto-agility}
+The shared secret established by the AKE must be known only to the two authenticated endpoints.
 
-Motivated by long deployment lifetimes, the AKE is required to support crypto agility, including modularity of COSE crypto algorithms and negotiation of preferred crypto algorithms for OSCORE and the AKE.
+A passive network attacker should never learn any session keys, even if it knows both endpoints' long-term keys.
 
-* The protocol shall support both pre-shared key and asymmetric key authentication. PAKE and post-quantum key exchange is out of scope, but may be supported in a later version.
-* The protocol shall allow multiple elliptic curves for asymmetric keys
-* The AKE shall support negotiation of all the COSE algorithms used in the AKE and that OSCORE supports. A successful negotiation shall result in the most preferred algorithms of one of the parties which are supported by the other.
-* The AKE shall support different AEAD/MAC algorithms for AKE and OSCORE
+An active attacker who has compromises the initiator or responder credential shall still not be able to compute past session keys (Perfect Forward Secrecy). These properties can be achieved e.g. with an ephemeral Diffie-Hellman key exchange.
 
+To mitigate against bad random number generators the AKE shall mandate randomness improvements such as {{I-D.irtf-cfrg-randomness-improvements}}.
 
-The AKE negotiation must be protected against downgrade attacks.
-\[Further detailing is requested.\]
+## Cryptographic Agility and Negotiation Integrity {#crypto-agility}
```

Comment:
The document was previously lacking a location for AKE integrity, even though we explicitly mentioned it in the Security Considerations. Some integrity properties (e.g. agreement on endpoint identities) comes from mutual authentication; others are related to what was negotiated and is noted here.